lib/api/src/types.rs

```diff
@@ -76,7 +76,7 @@ impl ValFuncRef for Val {
         let export = wasmer_engine::ExportFunction {
             // TODO:
             // figure out if we ever need a value here: need testing with complicated import patterns
-            import_init_function_ptr: None,
```

Comment:
Is the `TODO` above still valid?

Reply:
Yeah, I'm not 100% confident in this code which is why that comment is still there, I can investigate it later... I'm not aware of a good way to test this yet, I can investigate that though

Reply:
We should investigate. I bet is fine as None, but it will be good to verify